script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1

```diff
@@ -0,0 +1,110 @@
+<Page3>
+"...Well, what a surprise. I’m feeling incredibly well this morning...?"
+<Page4>
+Is it because I slept my way through yesterday? Or did Professor Arach’s IV drip work?
+Whatever it was, I feel really good. Too good in fact, that it’s making me a bit anxious.
+<Page5>
+There’s a knock on the door.
+After around ten seconds, Hisui entered the room.
+<Page6>
+"Pardon the intrusion."
+"Morning, Hisui. As always, thanks."
+<Page7>
+"Good morning, Shiki-sama. I see that you are already awake."
+"I slept like a log yesterday, so yeah. I’ll head down to the dining hall after I get changed. I’m feeling really hungry, so could you let Kohaku-san know to make my breakfast portions a bit bigger than usual?"
+<Page8>
+"I understand. I’m sure Nee-san will be delighted."
+Hisui bowed and exited the room.
+The way she carried herself felt a bit different than usual, to the point that I accidentally called out to her midway.
+<Page9>
+"Hisui, did something good happen?"
+"Yes. Your complexion has improved significantly, Shiki-sama. As your personal caretaker, there is no greater joy in this world than seeing you well."
+<Page10>
+So she said, and promptly exited the room.
+"――――――"
+Perhaps, to Hisui, those words were to be spoken as a natural part of her occupation. However, it felt like a sneak attack to me.
+"Ah, um... Yeah, well, thanks."
+As I scratched my flushing cheeks, I blabbered out something only I could hear.
+<Page11>
+"―――No, now’s not the time for this.
+I’m feeling good in a good while, I need to change quickly."
+<Page12>
+I switched my gears and shifted to preparing for my morning.
+It's Monday.
+The start of the week. A good time to start from scratch.
+<Page13>
+On today’s breakfast menu was Japanese.
+White rice and miso soup#.
+Spinach boiled in soy sauce with a squid base.
+And with triops shrimp as the main dish, it was a meal that went easy on the stomach.
+<Page14>
+"Here, please feel free to dig in.
+If you feel like having more, I can add another course for you, so please let me know."
+<Page15>
+"Ah... Sure..."
+A simple course it may be, but what on earth is this glowing <charm|a u r a  >radiating from this? Does Kohaku-san really not have a culinary background from some restaurant?
+<Page16>
+"Actually, no, this is more than enough. Thank you very much."
+I clapped my hands together, offered my gratitude for this lovely meal and started eating.
+My chopsticks advanced forward to make up for yesterday.
+Before I could even relish my meal, my bowl lay empty before me, side dishes exhausted.
+<Page17>
+"...I’m sorry, I didn’t savor this.
+...Um, could I please have seconds?"
+<Page18>
+"Of course. Please give me a moment, I’ll go refill your bowl."
+With that, she took my bowl and was on her way to the kitchen.
+<Page19>
+And while her back was turned, I spoke up,
+"Ah...Then could you-"
```

Comment:
> 「あ……それなら、その、」

I might expect this to be something closer to
> While you're in there―

But no matter what, needs cjk dash